src/meta/processors/admin/HBProcessor.cpp

```diff
@@ -46,9 +46,6 @@ void HBProcessor::process(const cpp2::HBReq& req) {
   }
 
   HostInfo info(time::WallClock::fastNowInMilliSec(), req.get_role(), req.get_git_info_sha());
-  if (req.version_ref().has_value()) {
-    info.version_ = *req.version_ref();
```

Comment:
Need you to delete the `version_` from `HostInfo` struct ?

Reply:
I haven’t deleted it yet, but it’s no used. Do I need to delete it?

Reply:
yeah

Reply:
done